homeassistant/helpers/area_registry.py

```diff
@@ -366,15 +368,13 @@ async def async_load(self) -> None:
         if data is not None:
             for area in data["areas"]:
                 assert area["name"] is not None and area["id"] is not None
-                normalized_name = normalize_name(area["name"])
```

Comment:
This looks like it might break when loading old area registry files from older versions that don’t have the key

Reply:
I'm sorry, I don't quite follow, what you are saying.
I'm removing code here, not adding.

Reply:
Nevermind, I misread it when I was looking on my mobile